src/k2/module/k23si/client/k23si_client.cpp

```diff
@@ -301,6 +313,17 @@ K23SIClient::K23SIClient(const K23SIClientConfig &) :
         sm::make_counter("abort_conflicts", abort_conflicts, sm::description("Total K23SI transactions aborted due to conflict"), labels),
         sm::make_counter("abort_too_old", abort_too_old, sm::description("Total K23SI transactions aborted due to retention window expiration"), labels),
         sm::make_counter("heartbeats", heartbeats, sm::description("Total K23SI transaction heartbeats sent"), labels),
+
+        sm::make_histogram("read_latency", [this]{ return _readLatency.getHistogram();}, sm::description("Latency of reads"), labels),
+        sm::make_histogram("write_latency", [this]{ return _writeLatency.getHistogram();}, sm::description("Latency of writes"), labels),
+        sm::make_histogram("partial_update_latency", [this]{ return _partialUpdateLatency.getHistogram();}, sm::description("Latency of writes"), labels),
+        sm::make_histogram("txn_latency", [this]{ return _txnLatency.getHistogram();}, sm::description("Latency of entire txns"), labels),
```

Comment:
we should rename this variable and description. This is not latency of entire txn. This is latency of beginTxn. The variable should be called _txnBeginLatency

Reply:
Done